paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/instrumentation/ResourcesCompatVisitorFactory.kt

```diff
@@ -73,10 +62,13 @@ internal abstract class ResourcesCompatVisitorFactory : AsmClassVisitorFactory<I
     }
 
     private companion object {
-      const val RESOURCES_COMPAT_CLASS_NAME = "androidx/core/content/res/ResourcesCompat"
       const val LOAD_FONT_METHOD_NAME = "loadFont"
       const val LOAD_FONT_METHOD_DESCRIPTOR =
         "(Landroid/content/Context;Landroid/content/res/Resources;Landroid/util/TypedValue;IILandroidx/core/content/res/ResourcesCompat\$FontCallback;Landroid/os/Handler;ZZ)Landroid/graphics/Typeface;"
     }
   }
+
+  internal companion object {
```

Comment:
nit: private

Reply:
nit: keep the 3 consts together, but move the `private companion` from the inner class to the outer class level